docs/docs/contributions/development/setting-up-pants.mdx

```diff
@@ -102,20 +102,19 @@ Pants sets up its development virtualenv at `~/.cache/pants/pants_dev_deps/<arch
 
 ### VSCode guide
 
+Install and enable the official `Flake8` and `Black Formatter` extensions from Microsoft.
```

Comment:
And EvenBetterToml and RustAnalyzer?

Those are only needed if you're updating Rust code, so 🤷🏽 

My bigger recommendation would be to replace this with committing an appropriate config into the repo, so it comes down on clone anyways.

Reply:
I'm sure there are many ways we could improve our VSCode support and documentation, but for now this is an improvement on the previous docs so I would like to merge it.